INSTALLATION.md

```diff
@@ -96,12 +96,16 @@ rails s
 
 ## Using Docker
 
+### Using Docker Compose
+
 You can use Docker to run Octobox in development.
 
 First, [install Docker](https://docs.docker.com/engine/installation/). If you've got run macOS or Windows, Docker for Mac/Windows makes this really easy.
 
 > If you have Windows Home Edition, you'll need to download and run [Docker Toolbox](https://www.docker.com/products/docker-toolbox).
 
+Second, download the `docker-compose.yml` file from [here](https://raw.githubusercontent.com/octobox/octobox/master/docker-compose.yml)
+
```

Comment:
Should we expect that the developer has cloned the repo here or not?

Reply:
depend if he is going to contribute to the development he should clone the repo, if he just wants to use it he can just download the file.

sadly most of the users clone the repo and build the image locally when they just need to download `docker-compose.yml` file and pull the images from docker hub.